ci-operator/step-registry/openshift-extended/test/stress/openshift-extended-test-stress-commands.sh

```diff
@@ -0,0 +1,596 @@
+    fi
+    echo "TEST_FILTERS: \"${hardcoded_filters};${TEST_FILTERS:-}\""
+    echo "FILTERS_ADDITIONAL: \"${FILTERS_ADDITIONAL:-}\""
+    test_filters="${hardcoded_filters};${TEST_FILTERS}"
+    if [[ -n "${FILTERS_ADDITIONAL:-}" ]]; then
+        echo "add FILTERS_ADDITIONAL into test_filters"
+        test_filters="${hardcoded_filters};${TEST_FILTERS};${FILTERS_ADDITIONAL}"
+    fi
+    echo "------handle test filter start------"
+    echo "${test_filters}"
+    handle_filters "${test_filters}"
+    echo "------handle test filter done------"
+
+    echo "------handle module filter start------"
+    echo "MODULE_FILTERS: \"${MODULE_FILTERS:-}\""
+    handle_module_filter "${MODULE_FILTERS}"
+    echo "------handle module filter done------"
+
+    echo "------------------the case selected------------------"
+    selected_case_num=$(cat ./case_selected|wc -l)
+    if [ "W${selected_case_num}W" == "W0W" ]; then
+        echo "No Case Selected"
+        if [ "W${FORCE_SUCCESS_EXIT}W" == "WnoW" ]; then
+            echo "do not force success exit"
+            exit 1
+        fi
+        echo "force success exit"
+        exit 0
+    fi
+    echo ${selected_case_num}
+    cat ./case_selected
+    echo "-----------------------------------------------------"
+
+    # failures happening after this point should not be caught by the Overall CI test suite in RP
+    touch "${ARTIFACT_DIR}/skip_overall_if_fail"
+    ret_value=0
+    set -x
+    remove_kubeadmin_user
+    if [ "W${TEST_PROVIDER}W" == "WnoneW" ]; then
+        extended-platform-tests run --max-parallel-tests ${TEST_PARALLEL} \
+        -o "${ARTIFACT_DIR}/extended.log" \
+        --timeout "${TEST_TIMEOUT}m" --junit-dir="${ARTIFACT_DIR}/junit" -f ./case_selected || ret_value=$?
+    else
+        extended-platform-tests run --max-parallel-tests ${TEST_PARALLEL} \
+        --provider "${TEST_PROVIDER}" -o "${ARTIFACT_DIR}/extended.log" \
+        --timeout "${TEST_TIMEOUT}m" --junit-dir="${ARTIFACT_DIR}/junit" -f ./case_selected || ret_value=$?
+    fi
+    set +x
+    set +e
+    rm -fr ./case_selected
+    echo "try to handle result"
+    handle_result
+    echo "done to handle result"
+    if [ "W${ret_value}W" == "W0W" ]; then
+        echo "success"
+    else
+        echo "fail"
+    fi
+    # summarize test results
+    echo "Summarizing test results..."
+    failures=0 errors=0 skipped=0 tests=0
+    [[ -e "${ARTIFACT_DIR}" ]] || exit 0
+    grep -r -E -h -o 'testsuite.*tests="[0-9]+"' "${ARTIFACT_DIR}" | tr -d '[A-Za-z=\"_]' > /tmp/zzz-tmp.log
+    while read -a row ; do
+        # if the last ARG of command `let` evaluates to 0, `let` returns 1
+        let errors+=${row[0]} failures+=${row[1]} skipped+=${row[2]} tests+=${row[3]} || true
+    done < /tmp/zzz-tmp.log
+
+    TEST_RESULT_FILE="${ARTIFACT_DIR}/test-results.yaml"
+    cat > "${TEST_RESULT_FILE}" <<- EOF
+openshift-extended-test-stress:
```

Comment:
@kuiwang02 This file is copied from https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-extended/test/openshift-extended-test-commands.sh, the only change I did is this line, added '-stress' to the output. 
Please let me know if there is any other place I have to make change.